steps/codified-security-bitrise/1.0.0/step.yml

```diff
@@ -0,0 +1,53 @@
+title: Codified Security
+summary: This step uploads your app to Codified Security for automated mobile app
+  security testing
+description: This step uploads your app to Codified Security for automated mobile
+  app security testing
+website: https://codifiedsecurity.com
+published_at: 2017-04-19T19:05:03.975322686+02:00
+source:
+  git: https://bitbucket.org/codifiedldn/codified-security-bitrise.git
+  commit: b2d6cbb641d8f14e1f3632328cf56c8c857dc02a
+project_type_tags:
+- ios
+- android
+- xamarin
+type_tags:
+- android
+- apk
+- ios
+- swift
+- ipa
+- xamarin
+- xamarin.android
+- xamarin.ios
+- security
+- security-testing
+- vulnerabilities
+- mobile-app-security
+- owasp-mobile-top-10
+- security-checklist
+- static-analysis
+- owasp
+- penetration
+- testing
+- scan
+- native
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
```

Comment:
For me it makes sense to just ignore instead of fail whole build - this step isn't affects app builds 'just' runs a security analysis on an app. But if you think it shouldn't work like that then of course I'll change it.